components/gateway/internal/auditlog/service.go

```diff
@@ -35,10 +37,15 @@ func (sink *Sink) Log(request, response string, claims proxy.Claims) error {
 		Claims:   claims,
 	}
 
+	log.Printf("Attempting to register auditlog message %+v for processing to the queue (size=%d, capacity=%d)\n",
+		msg, len(sink.logsChannel), cap(sink.logsChannel))
+
 	select {
 	case sink.logsChannel <- msg:
+		log.Printf("Successfully registered auditlog message %+v for processing to the queue (size=%d, capacity=%d)\n",
+			msg, len(sink.logsChannel), cap(sink.logsChannel))
 	case <-time.After(sink.timeout):
-		return errors.New("Cannot write to the channel")
+		return errors.New(fmt.Sprintf("Failed to write message %+v to the channel", msg))
```

Comment:
The message should not be printed as it might contain sensitive data. (the same applies to auditlog anyway :D, but...)  Leave this error as it was.

Reply:
Good point :D 👍🏼

Reply:
The same applies to other places where there is such msg print